### PR TITLE
continuos integration setup

### PR DESCRIPTION
**Pull request by Millionhat**
Setup for travis test and success of first trial
image

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+directories:
+- node_modules
+notifications:
+email: false
+branches:
+except:
+- "/^v\\d+\\.\\d+\\.\\d+$/"
+script:
+  - npm test
+
```
Comment (holgiosalos) [inline .travis.yml]: EOF

**Comment by Millionhat** [comment]
it should be EOF

File: package.json [hunk 2]
```diff
@@ -28,13 +29,15 @@
...
     "eslint": "^7.31.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-cypress": "^2.11.3",
     "eslint-plugin-import": "^2.23.4",
     "ts-loader": "^9.2.3",
     "typescript": "^4.3.5",
     "webpack": "^5.46.0"
+  },
+  "dependencies": {
+    "cypress": "^8.0.0"
```
Comment (manuelq12) [inline package.json]: This shouldn't be dependencies, should be dev-dependencies

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Millionhat) [reply package.json]: when i changed it to dev-dependencies the last class the tests would not run

Comment (holgiosalos) [reply package.json]: yes, it was something related with the linter, could you move to dev dependency and see if it fails again? If it does not fail, please commit that change, if it fails, then let us know to approve this PR :)

**Review by holgiosalos — APPROVED** [review]

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**